pyiode/iode_database/abstract_database.pyx

```diff
@@ -469,6 +473,9 @@ cdef class _AbstractDatabase:
 
         self.abstract_db_ptr.copy_into(input_files.encode(), objects_names.encode())
 
+    # GDM>:
+    # * same comment as above (i.e. rename to merge_from?)
+    # * what is the difference between merge_into and copy_to? The doctests do not help me understand the difference.
```

Comment:
Honnestly, the french documentation about `WsCopy` and `WsMerge` is not clear. I don't know the difference.
@jmpplan Could you explain the difference between `WsCopy` and `WsMerge` please?